src/main/java/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -328,6 +335,9 @@ else if (!record.getReadFailsVendorQualityCheckFlag()) {
                 }
             }
 
+            private double divideIfYouCan(double numerator, double denominator){
```

Comment:
Is this different than `MathUtil.divide()`?

Reply:
yes...but that is still good enough for this purpose. thanks!